Undefined symbol during link of tf_conversions

Doing a build from source against today's version of tf_conversions, I'm getting this linker error:

```
Scanning dependencies of target tf_conversions
[ 50%] [100%] Building CXX object CMakeFiles/tf_conversions.dir/src/tf_kdl.cpp.o
Building CXX object CMakeFiles/tf_conversions.dir/src/tf_eigen.cpp.o
Linking CXX shared library /Users/tlau/ros/devel_isolated/tf_conversions/lib/libtf_conversions.dylib
Undefined symbols for architecture x86_64:
  "KDL::Vector::Normalize(double)", referenced from:
      KDL::Rot(KDL::Vector const&) in tf_kdl.cpp.o
  "KDL::Rotation::Quaternion(double, double, double, double)", referenced from:
      tf::quaternionTFToKDL(tf::Quaternion const&, KDL::Rotation&) in tf_kdl.cpp.o
  "KDL::operator*(KDL::Rotation const&, KDL::Rotation const&)", referenced from:
      KDL::operator*(KDL::Frame const&, KDL::Frame const&) in tf_kdl.cpp.o
      KDL::addDelta(KDL::Rotation const&, KDL::Vector const&, double) in tf_kdl.cpp.o
  "KDL::Rotation::GetQuaternion(double&, double&, double&, double&) const", referenced from:
      tf::quaternionKDLToTF(KDL::Rotation const&, tf::Quaternion&) in tf_kdl.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [/Users/tlau/ros/devel_isolated/tf_conversions/lib/libtf_conversions.dylib] Error 1
make[1]: *** [CMakeFiles/tf_conversions.dir/all] Error 2
make: *** [all] Error 2
<== Failed to process package 'tf_conversions': 
  Command '/opt/ros/hydro/env.sh make -j4 -l4' returned non-zero exit status 2
```

I'm building according to [hydro installation instructions](http://wiki.ros.org/hydro/Installation/OSX/Homebrew/Source) on Mac OSX.

Any ideas?

Here is my stack trace : 
Undefined symbols for architecture x86_64:
  "KDL::Vector::Normalize(double)", referenced from:
      KDL::Rot(KDL::Vector const&) in tf_kdl.cpp.o
  "KDL::Rotation::Quaternion(double, double, double, double)", referenced from:
      tf::quaternionTFToKDL(tf::Quaternion const&, KDL::Rotation&) in tf_kdl.cpp.o
  "KDL::operator_(KDL::Rotation const&, KDL::Rotation const&)", referenced from:
      KDL::operator_(KDL::Frame const&, KDL::Frame const&) in tf_kdl.cpp.o
      KDL::addDelta(KDL::Rotation const&, KDL::Vector const&, double) in tf_kdl.cpp.o
  "KDL::Rotation::GetQuaternion(double&, double&, double&, double&) const", referenced from:
      tf::quaternionKDLToTF(KDL::Rotation const&, tf::Quaternion&) in tf_kdl.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: **\* [/Users/chradja/ros_catkin_ws/devel_isolated/tf_conversions/lib/libtf_conversions.dylib] Error 1
make[1]: **\* [CMakeFiles/tf_conversions.dir/all] Error 2
make: **\* [all] Error 2

--- a/tf_conversions/CMakeLists.txt
+++ b/tf_conversions/CMakeLists.txt
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 2.8)
 project(tf_conversions)
 
 find_package(Eigen REQUIRED)
+find_package(orocos_kdl REQUIRED)
 find_package(catkin REQUIRED geometry_msgs kdl_conversions tf)
 
 catkin_python_setup()
@@ -9,11 +10,11 @@ catkin_python_setup()
 catkin_package(
   INCLUDE_DIRS include
   LIBRARIES ${PROJECT_NAME}
-  DEPENDS Eigen
+  DEPENDS Eigen orocos_kdl
   CATKIN_DEPENDS geometry_msgs kdl_conversions tf
 )
 
-include_directories(SYSTEM ${EIGEN_INCLUDE_DIRS})
+include_directories(SYSTEM ${EIGEN_INCLUDE_DIRS} ${orocos_kdl_INCLUDE_DIRS})
 include_directories(include ${catkin_INCLUDE_DIRS})
 
 # Needed due to no full filename in orocos_kdl pkg-config export
@@ -24,17 +25,17 @@ add_library(${PROJECT_NAME}
   src/tf_eigen.cpp
 )
 add_dependencies(${PROJECT_NAME} geometry_msgs_gencpp)
-target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES} ${orocos_kdl_LIBRARIES})
 
 # Tests
 if(CATKIN_ENABLE_TESTING)
 
 
 catkin_add_gtest(test_eigen_tf test/test_eigen_tf.cpp)
-target_link_libraries(test_eigen_tf ${PROJECT_NAME})
+target_link_libraries(test_eigen_tf ${PROJECT_NAME} ${orocos_kdl_LIBRARIES})
 
 catkin_add_gtest(test_kdl_tf test/test_kdl_tf.cpp)
-target_link_libraries(test_kdl_tf ${PROJECT_NAME})
+target_link_libraries(test_kdl_tf ${PROJECT_NAME} ${orocos_kdl_LIBRARIES})
 
 catkin_add_nosetests(test/posemath.py)
 

--- a/tf_conversions/package.xml
+++ b/tf_conversions/package.xml
@@ -25,11 +25,13 @@ the next major release cycle (see roadmap).
   <build_depend>geometry_msgs</build_depend>
   <build_depend>kdl_conversions</build_depend>
   <build_depend>tf</build_depend>
+  <build_depend>orocos_kdl</build_depend>
 
   <run_depend>eigen</run_depend>
   <run_depend>geometry_msgs</run_depend>
   <run_depend>kdl_conversions</run_depend>
   <run_depend>tf</run_depend>
+  <run_depend>orocos_kdl</run_depend>
 
   <export>
     <rosdoc config="rosdoc.yaml" />

